tasks/modules.py

```diff
@@ -133,7 +133,7 @@ def dependency_path(self, agent_version):
     "test/new-e2e": GoModule("test/new-e2e", condition=lambda: False, should_tag=False),
     "test/fakeintake": GoModule("test/fakeintake", independent=True, should_tag=False),
     "pkg/obfuscate": GoModule("pkg/obfuscate", independent=True),
-    "pkg/trace": GoModule("pkg/trace", independent=True),
+    "pkg/trace": GoModule("pkg/trace", independent=False),
```

Comment:
@knusbaum does `pkg/trace` need to be an independent module? If it does I can probably move stuff around, but the way it might be related to the rest of the codebase I'm not sure it makes sense. Would love to understand this further.

If it *does* need to be independent, that's OK I can probably move some code around, just want to fully understand this.

Reply:
Adding here for contextual reference: the module must remain independent as it is imported by the otel collector, and we can thus not have references to `github.com/DataDog/datadog-agent`.